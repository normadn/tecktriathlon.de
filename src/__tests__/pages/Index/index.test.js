import React from "react";
import IndexPage from "../../../pages/index";
import { shallow } from "enzyme";

const data = {
  site: {
    siteMetadata: {
      netlifyEnv: "deploy-preview"
    }
  }
};

describe("IndexPage", () => {
  it("renders correctly", () => {
    expect(shallow(<IndexPage data={data} />)).toMatchSnapshot();
  });
});
