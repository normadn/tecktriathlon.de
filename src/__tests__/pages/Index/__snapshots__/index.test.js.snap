// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<ApolloProvider
  client={
    DefaultClient {
      "cache": InMemoryCache {
        "addTypename": true,
        "cacheKeyRoot": CacheKeyNode {
          "children": null,
          "key": null,
        },
        "config": Object {
          "addTypename": true,
          "dataIdFromObject": [Function],
          "fragmentMatcher": HeuristicFragmentMatcher {},
          "resultCaching": true,
        },
        "data": DepTrackingCache {
          "data": Object {},
          "depend": [Function],
        },
        "maybeBroadcastWatch": [Function],
        "optimistic": Array [],
        "silenceBroadcast": false,
        "storeReader": StoreReader {
          "cacheKeyRoot": CacheKeyNode {
            "children": null,
            "key": null,
          },
          "executeSelectionSet": [Function],
          "executeStoreQuery": [Function],
          "keyMaker": QueryKeyMaker {
            "cacheKeyRoot": CacheKeyNode {
              "children": null,
              "key": null,
            },
            "perQueryKeyMakers": Array [],
          },
        },
        "storeWriter": StoreWriter {},
        "typenameDocumentCache": Array [],
        "watches": Array [],
      },
      "clientAwareness": Object {},
      "defaultOptions": Object {},
      "disableNetworkFetches": false,
      "link": ApolloLink {
        "request": [Function],
      },
      "mutate": [Function],
      "query": [Function],
      "queryDeduplication": true,
      "reFetchObservableQueries": [Function],
      "resetStore": [Function],
      "resetStoreCallbacks": Array [],
      "ssrMode": false,
      "store": DataStore {
        "cache": InMemoryCache {
          "addTypename": true,
          "cacheKeyRoot": CacheKeyNode {
            "children": null,
            "key": null,
          },
          "config": Object {
            "addTypename": true,
            "dataIdFromObject": [Function],
            "fragmentMatcher": HeuristicFragmentMatcher {},
            "resultCaching": true,
          },
          "data": DepTrackingCache {
            "data": Object {},
            "depend": [Function],
          },
          "maybeBroadcastWatch": [Function],
          "optimistic": Array [],
          "silenceBroadcast": false,
          "storeReader": StoreReader {
            "cacheKeyRoot": CacheKeyNode {
              "children": null,
              "key": null,
            },
            "executeSelectionSet": [Function],
            "executeStoreQuery": [Function],
            "keyMaker": QueryKeyMaker {
              "cacheKeyRoot": CacheKeyNode {
                "children": null,
                "key": null,
              },
              "perQueryKeyMakers": Array [],
            },
          },
          "storeWriter": StoreWriter {},
          "typenameDocumentCache": Array [],
          "watches": Array [],
        },
      },
      "version": "2.4.8",
      "watchQuery": [Function],
    }
  }
>
  <div
    css={
      Object {
        "overflowX": "hidden",
      }
    }
  >
    <HelmetWrapper
      defer={true}
      encodeSpecialCharacters={true}
      link={
        Array [
          Object {
            "href": "test-file-stub",
            "rel": "shortcut icon",
          },
        ]
      }
      meta={
        Array [
          Object {
            "content": "QHacks is Queen’s University’s annual hackathon, happening February 1st - 3rd, 2019.",
            "name": "description",
          },
          Object {
            "content": "Hackathon, Queen's University, Kingston, QHacks, Queensu",
            "name": "keywords",
          },
          Object {
            "content": "width=device-width, initial-scale=1.0",
            "name": "viewport",
          },
          Object {
            "content": "website",
            "property": "og:type",
          },
          Object {
            "content": "https://qhacks.io/",
            "property": "og:url",
          },
          Object {
            "content": "QHacks | Queen's University | Winter 2019",
            "property": "og:title",
          },
          Object {
            "content": "test-file-stub",
            "property": "og:image",
          },
          Object {
            "content": "QHacks is back for round four! We're bringing in students from all over North America to come together for a wild 36 hours - designing, developing, demoing, cup-stacking, and foosball-playing at our home base at Queen's University. Whether you're a first timer or a seasoned veteran, QHacks definitely has something to satisfy your interests. This weekend is about inclusiveness for all disciplines - students from any level of skill, from any field of study are encouraged to bring their innovative ideas for a weekend of creative problem solving! The only prerequisites for this weekend are a love for tech and drive to learn.",
            "property": "og:description",
          },
          Object {
            "content": "QHacks",
            "property": "og:site_name",
          },
          Object {
            "content": "en_US",
            "property": "og:locale",
          },
        ]
      }
      title="QHacks | Queen's University | Winter 2019"
    >
      <html
        lang="en"
      />
    </HelmetWrapper>
    <div>
      <Landing />
      <MailingListSignup />
      <About />
      <HackerTestimonials />
      <Speakers />
      <EventSchedule />
      <Sponsors />
      <FAQs />
      <Podcast />
      <Footer />
    </div>
  </div>
</ApolloProvider>
`;
